controllers/admin/AdminSearchConfController.php

```diff
@@ -181,6 +181,49 @@ public function __construct()
                             ),
                         ),
                     ),
+                    'PS_SEARCH_FUZZY' => array(
+                        'title' => $this->trans('Fuzzy Search', array(), 'Admin.Shopparameters.Feature'),
+                        'validation' => 'isBool',
+                        'cast' => 'intval',
+                        'type' => 'bool',
+                        'desc' => $this->trans(
+                                'By default, fuzzy search is enabled and spelling error are allowed. To search “bird”, you can to enter “burd”, “bard”, “beerd” etc...',
+                                array(),
+                                'Admin.Shopparameters.Help'
+                            ) . '<br/>' .
+                            $this->trans(
+                                'With this option disabled, to get good result, you need get exact writting contained in the word.',
+                                array(),
+                                'Admin.Shopparameters.Help'
+                            ),
+                        'hint' => array(
+                            $this->trans(
+                                'Enable spelling errors during search.',
+                                array(),
+                                'Admin.Shopparameters.Help'
+                            ),
+                            $this->trans(
+                                'Fuzzy search will check if a close word of the search word exist in indexed words in database. This will be very resource-consuming.',
+                                array(),
+                                'Admin.Shopparameters.Help'
+                            ),
+                        ),
+                    ),
+                    'PS_SEARCH_FUZZY_MAX_LOOP' => array(
+                        'title' => $this->trans(
+                            'Maximum words with spelling error allowed by search',
+                            array(),
+                            'Admin.Shopparameters.Feature'
+                        ),
+                        'hint' => $this->trans(
+                            'A fuzzy search is very resource-consuming. More words you have to solve, more a search could take time.',
```

Comment:
```suggestion
                            'Note that this option is time-consuming: the more you check, the longer it searches.',
```

Reply:
Is it time-consuming or resource-consuming? I think that there is a slight difference between the two.
🤔

Reply:
Done

Reply:
@LouiseBonnard 
I would choose "resource consuming", because the time it will take really depends on the shop's infrastructure.

Reply:
Done !

Reply:
Nice, thank you, it's perfectly clear now. What about _Note that this option is resource-consuming: the more you search, the longer it takes._ instead? I think it is even more explicit.

Reply:
Done